invenio_app_rdm/theme/webpack.py

```diff
@@ -44,7 +45,7 @@
                 'react-dnd-html5-backend': '^11.1.0',
                 'react-dropzone': "^11.0.0",
                 "react-i18next": "^11.11.0",
-                'react-invenio-deposit': '^0.18.0',
+                'react-invenio-deposit': '^0.18.1',
```

Comment:
no need to increase? It will still get the latest. I removed all the non-zero-versions some time ago.